### PR TITLE
API Improvements

### PR DESCRIPTION
**Pull request by mrexodia**
- Moved encoder/decoder related stuff to the `Dotx64Dbg.Disassembly` namespace (name might not be ideal, but I think it's reasonably discoverable)
- Added some aliases to improve discoverability

### CODE REVIEW

**Comment by ZehMatt** [comment]
What is the reason to move it to a different namespace?

**Comment by mrexodia** [comment]
Mostly because `Label` is colliding with the Label concept inside x64dbg. It also makes it more difficult to discover the API. Now you can write `DotX64Dbg.Disassembly.` and quickly see the related classes.

The only downside seems to be that you have to add a new `using` statement in existing scripts.

File: src/Bindings/Instruction.Decoder.cpp [hunk 1]
```diff
@@ -230,4 +230,19 @@ namespace Dotx64Dbg
...
 
     };
+
+    // Alias for discoverability
+    public ref class Disassembler
+    {
+    private:
+        Disassembler()
+        {
+        }
```
Comment (ZehMatt) [inline src/Bindings/Instruction.Decoder.cpp:240]:
```suggestion
        Disassembler() = default;
```

Comment (ZehMatt) [reply src/Bindings/Instruction.Decoder.cpp]: That's apparently not supported, sorry abut that.

**Review by ZehMatt — APPROVED** [review]

**Pull request merged**